deeprank_gnn/models/graph.py

```diff
@@ -1,11 +1,12 @@
+from typing import Dict
 from uuid import uuid4
 
 from networkx import Graph as NetworkxGraph
 
 class Graph(NetworkxGraph):
     "this is a graph just like in networkx, but with an id and associated target values"
 
-    def __init__(self, id_=None, targets=None):
+    def __init__(self, id_: str = None, targets: Dict[str, float] = None):
```

Comment:
Shouldn't the optional arguments be set to type Optional?

Reply:
Yes! I changed this everywhere in the models folder.